maistra/bazelrc

```diff
@@ -21,6 +21,9 @@ build --host_javabase=@bazel_tools//tools/jdk:absolute_javabase --define=ABSOLUT
 build --force_pic=true
 build --verbose_failures
 
+# release build: define NDEBUG 
+build:release --copt -DNDEBUG
```

Comment:
I don't get this.

We are now building with `--config=debug`. How this will fix it?

Reply:
> We are now building with --config=debug. How this will fix it?

We are doing release builds using debug config? Where are we doing this? I don't think we should...